docs/circuit_cutting/explanation/index.rst

```diff
@@ -1,5 +1,46 @@
-###########################
-Circuit Cutting Explanation
-###########################
+###################################################
+Explanatory material for the circuit cutting module
+###################################################
 
-Circuit cutting is a technique to decompose a quantum circuit into smaller circuits, whose results can be knitted together to reconstruct the original circuit output.
+Overview of circuit cutting
+---------------------------
+Circuit cutting is a technique to increase the size of circuits we can run on quantum hardware at the cost of an additional sampling overhead. A larger quantum circuit can be decomposed by cutting its gates and wires into smaller circuits, which can be executed within the constraints of available quantum hardware. The results of these smaller circuits are combined to reconstruct the outcome of the original problem. Circuit cutting can also be used to engineer gates between distant qubits which would otherwise require a large swap overhead.
+
+Quasiprobability Decomposition (QPD)
+------------------------------------
+Quasiprobability decomposition is a technique which can be used to simulate noise-free quantum gates using only noisy, local operations (LO). This is often referred to as circuit cutting. One could cut a non-local gate and simulate it using only local operations. This is referred to as a "gate cut" or "space-like cut". One could also imagine cutting a single-qubit identity gate and simulating it using local operations on either side of the cut. The other side of the cut would be simulated by introducing a new qubit into the circuit and moving remaining operations after the cut identity gate to the new qubit. This is referred to as a "wire cut" or "time-like cut". The cost of conducting these techniques is an exponential sampling overhead. If no real-time classical communication is available between qubits of the cut gate or wire, gate cuts incur a sampling overhead of O(9\ :sup:`n`), and wire cuts incur a sampling overhead of O(16\ :sup:`n`), where n is the total number of cuts. If real-time communication is available (i.e. dynamic circuits), the sampling overhead for both gate and wire cuts may be reduced to O(4\ :sup:`n`) [`1 <https://arxiv.org/abs/2205.00016>`_] [`3 <https://arxiv.org/abs/2302.03366>`_]; however, support for circuit cutting with classical communication (LOCC) is not yet supported in CKT.
+
+For more detailed information on this technique, refer to the paper, Error mitigation for short-depth quantum circuits [`4 <https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.119.180509>`_].
```

Comment:
I like to use arXiv links for these things, because the arXiv paper is available without a subscription, and it always links to the real paper.  An alternative could be to include both links.  Same comment where the reference is listed at the end of this document.

https://arxiv.org/abs/1612.02058